src/css.js

```diff
@@ -113,6 +113,13 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( !support.reliableDimensionsInFullscreenMode && window.top !== window.self &&
+		document.msFullscreenElement ) {
+		val = Math.round( elem.getBoundingClientRect()[name] * 100 );
```

Comment:
Spaces are needed after `[` and before `]` as well. We should get this JSCS update ASAP because it's hard to catch all these cases manually...